compute_sdk/tests/unit/test_client.py

```diff
@@ -769,6 +798,16 @@ def test_client_logout_with_login_manager():
     assert mock_lm.logout.called
 
 
+def test_client_logout_with_authorizer_warning(mocker):
+    mock_log = mocker.patch(f"{_MOCK_BASE}logger")
+    mock_authorizer = mock.Mock(spec=GlobusAuthorizer)
+    client = gc.Client(do_version_check=False, authorizer=mock_authorizer)
+    client.logout()
+    a, _ = mock_log.warning.call_args
+    assert "Logout not supported" in a[0]
+    assert type(mock_authorizer).__name__ in a[0]
+
```

Comment:
What's the reason for checking the type is in the warning?  (As in, suggest putting that in the `assert ..., "message"`.)

Reply:
It just adds a bit of clarity to the warning. I added a message to note this.